hops/experiment.py

```diff
@@ -56,7 +56,7 @@ def launch(map_fun, args_dict=None, name='no-name', local_logdir=False, descript
     >>> def train_nn():
```

Comment:
below (line 135) you also renamed it in the example to `train_fn`. doesn't really matter what it's called but let's be consistent.

Reply:
Updated the examples to be consistent